logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/call/CallsScope.kt

```diff
@@ -54,4 +58,10 @@ class CallsScope(
     val updateVideoState: UpdateVideoStateUseCase get() = UpdateVideoStateUseCase(callManager, callRepository)
 
     val setVideoPreview: SetVideoPreviewUseCase get() = SetVideoPreviewUseCase(flowManagerService)
+
+    val turnLoudSpeakerOffUseCase: TurnLoudSpeakerOffUseCase get() = TurnLoudSpeakerOffUseCase(mediaManagerService)
+
+    val turnLoudSpeakerOnUseCase: TurnLoudSpeakerOnUseCase get() = TurnLoudSpeakerOnUseCase(mediaManagerService)
+
+    val observeSpeakerUseCase: ObserveSpeakerUseCase get() = ObserveSpeakerUseCase(mediaManagerService)
```

Comment:
To keep consistency we can remove the `UseCase` suffix from them.

```suggestion
    val turnLoudSpeakerOff: TurnLoudSpeakerOffUseCase get() = TurnLoudSpeakerOffUseCase(mediaManagerService)

    val turnLoudSpeakerOn: TurnLoudSpeakerOnUseCase get() = TurnLoudSpeakerOnUseCase(mediaManagerService)

    val observeSpeaker: ObserveSpeakerUseCase get() = ObserveSpeakerUseCase(mediaManagerService)
```